notebooks/6_Putting_it_All_Together.ipynb

```diff
@@ -0,0 +1,1002 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook, we will tie in many of the pieces covered in this tutorial.\n",
+    "\n",
+    "First, we will take a materials dataset, in the form of cartesian coordinates, and translate it into SOAP descriptors using `librascal`, as was covered briefly in [**X_ImportingData.ipynb**](X_ImportingData.ipynb).\n",
+    "\n",
+    "Next, we'll reduce the feature space of these SOAP vectors using PCov-CUR, which is covered in [**5_CUR.ipynb**](5_CUR.ipynb). \n",
+    "\n",
+    "From here, we'll select an _active_ set of samples using PCov-CUR and use them to build a set of sparse kernels, as was introduced [**4_SparseKernelMethods.ipynb**](4_SparseKernelMethodsipynb), and use this sparse kernel to build a `SparseKPCovR` map.\n",
+    "\n",
+    "Finally, we will port this `SparseKPCovR` into a `chemiscope` visualization (covered in [**1_LinearMethods.ipynb**](1_LinearMethods.ipynb)), which can be loaded on [chemiscope.org](chemiscope.org).\n",
+    "\n",
+    "With each of the sections, we'll provide a \"fast-forward\" cell, where you can skip the section by un-commenting and executing the cell. For example, if you have pre-computed your SOAP vectors and would like to simply import them, skip section 1 and execute the cell under `1.1 Skip Computing SOAPS`."
```

Comment:
Maybe add a section on "dependencies you'll need", giving a `pip install` command? Something like `pip install sklearn numpy matplotlib scipy skcosmo ase chemiscope git+https://github.com/cosmo-epfl/librascal#e78f1c69`